system/CodeIgniter.php

```diff
@@ -967,6 +965,8 @@ public function storePreviousURL($uri)
     /**
      * Modifies the Request Object to use a different method if a POST
      * variable called _method is found.
+     *
+     * @deprecated 4.2
```

Comment:
Yes, because of the handling of the spoofing method inside Request::getMethod() , there is no point in this method. But I left it, otherwise it will suddenly be BC.